### ISSUE TITLE
[doc] 6.4.3.1  Deploy Ceph Monitors and Ceph Managers

### ISSUE THREAD

**Issue by tspmelo** [SUSE/doc-ses#318]
6.4.3.1  Deploy Ceph Monitors and Ceph Managers

https://susedoc.github.io/doc-ses/master/html/ses-deployment/deploy-cephadm.html#deploy-cephadm-day2-service-mon

After reading:
`If there are five or more nodes in the cluster, we recommend deploying five MONs`

```
To deploy new MONs, run:
ceph orch apply mon SHORT_MON_HOST_NAME
```

```
If MONs or MGRs are not on the same subnet, you need to append the subnet address. For example:

ceph orch apply mon ses-min1:10.1.2.0/24
ceph orch apply mon ses-min2:10.1.5.0/24
ceph orch apply mgr ses-min1:10.1.2.0/24
ceph orch apply mgr ses-min2:10.1.5.0/24
```

I got the idea that I should run `ceph orch apply mon SHORT_MON_HOST_NAME` to all nodes in my cluster and all would have mon deployed.
But what I got was that only the last node I ran the cmd was the `mon`, all the reamaing got nothing.
I think the documentation should mention that running `ceph orch apply mon SHORT_MON_HOST_NAME` will deploy to the node and remove from the previous.

At least I think the subnet example should not contain cmds for multiple nodes.

**Comment by tbazant** [comment]
@sebastian-philipp can you confirm @tspmelo 's findings? what is then the correct MON/MGR assignment?

**Comment by tbazant** [comment]
This was already fixed by #333

**Comment by tbazant** [comment]
fixed elsewhere, closing
